cns/restserver/util.go

```diff
@@ -165,8 +165,7 @@ func (service *HTTPRestService) saveNetworkContainerGoalState(req cns.CreateNetw
 		case cns.AzureFirstParty:
 			fallthrough
 		case cns.WebApps: // todo: Is WebApps an OrchastratorType or ContainerType?
-			var podInfo cns.KubernetesPodInfo
```

Comment:
Again, I dont think we can change this. Unmarshal will fail as OrchestratorContext is only pod name + podns